samples/cpp/example_cmake/Makefile

```diff
@@ -0,0 +1,12 @@
+CXX ?= g++
+
+CXXFLAGS += -c -Wall `pkg-config --cflags opencv`
+LDFLAGS += `pkg-config --libs --static opencv`
```

Comment:
I think it would be better to use `$(shell ...)` here, to only query this stuff once per Makefile. It's not very important for _this_ Makefile, but it would set a better example.